ServerStatisticsConsumer/TopicConsumer.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Text;
+using Newtonsoft.Json;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using ServerConfigurations.Configurations;
+using ServerStatisticsConsumer.Services;
+using ServerStatisticsService;
+using SignalRService;
+
+namespace ServerStatisticsConsumer
+{
+    public static class TopicConsumer
+    {
+        public static async Task Consume(IModel channel , IMongoDB mongoDB 
+            , AnomalyDetectionConfig configThreshold , ServerAlertService serverAlertService)
+        {
+            channel.ExchangeDeclare("topic-ServerStatistics-exchange", ExchangeType.Topic);
+            channel.QueueDeclare("topic-ServerStatistics-queue",
+               durable: true,
+               exclusive: false,
+            autoDelete: false,
+            arguments: null);
+
+            channel.QueueBind("topic-ServerStatistics-queue", "topic-ServerStatistics-exchange", "ServerStatistics.*");
+
+            var alertCalculation = new AlertCalculations(configThreshold, serverAlertService);
```

Comment:
based on the previous comment, this needs to be passed to the constructor instead of resolving it from within the method